trackpy/feature.py

```diff
@@ -207,6 +208,7 @@ def _refine(raw_image, image, radius, coords, max_iterations,
     GOOD_ENOUGH_THRESH = 0.005
 
     ndim = image.ndim
+    anisotropic = radius[1:] != radius[:-1]
```

Comment:
https://github.com/soft-matter/trackpy/pull/239/files#diff-8caca4dcbf45ed9519ea2c65b2c391e6R239 is the line I was looking at.  

If the length is 2 than this will work correctly with ndarrays (as length 1 arrays are apparently unambiguous).